### PR TITLE
Fix cross-page export table mapping issue

### PR DESCRIPTION
**Pull request by tklengyel**
Fixes issue #260.

### CODE REVIEW

File: libvmi/os/windows/peparse.c [hunk 1]
```diff
@@ -516,7 +516,25 @@ peparse_get_export_table(
...
     if (nbytes != sizeof(struct export_table)) {
         dbprint(VMI_DEBUG_PEPARSE, "--PEParse: failed to map export header\n");
-        return VMI_FAILURE;
+
+        /*
+         * Sometimes Windows maps the export table on page-boundaries,
+         * such that the first export_flags field (which is reserved) is.
+         * not actually accessible (the page is not mapped). See Issue #260.
+         */
+        if (export_header_va >> 12 != (export_header_va+4) >> 12) {
```
Comment (awsaba) [inline libvmi/os/windows/peparse.c]: Is there anything to check that the addresses here are 4-byte aligned?  If not, maybe something like `(export_header_va + 4) & ~(0xfff)` could be used to include that as a sanity check?

Comment (tklengyel) [inline libvmi/os/windows/peparse.c]: It's not really the alignment that matters - it's whether the struct is is on a page-boundary with the first uint32_t mapped into another page.

Comment (awsaba) [inline libvmi/os/windows/peparse.c]: If the special case is "export_header_va is exactly 4 bytes below a page boundary", then that's what the check should be for, not "4 or less".

Comment (tklengyel) [inline libvmi/os/windows/peparse.c]: @awsaba: sure. The check you propose is a bit more restrictive but it does match what we currently see so I'll make the adjustment.

**Pull request merged**